serializers/web/pg_firewall.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Serializers::Web::PgFirewall < Serializers::Base
```

Comment:
How about Serializers::Web::PostgresFirewallRule? It is bit longer, but I think it matches better with rest.